frontend/appflowy_flutter/lib/plugins/document/presentation/editor_page.dart

```diff
@@ -7,6 +7,10 @@ import 'package:appflowy_editor/appflowy_editor.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 
+import 'editor_plugins/calendar/calendar_node_widget.dart';
+import 'editor_plugins/calendar/calendar_view_menu_item.dart';
+import 'editor_plugins/calendar/calendar_menu_item.dart';
```

Comment:
Perhaps we could export these three files in `appflowy_flutter/lib/plugins/document/presentation/editor_plugins/plugins.dart` like the other database views? That way we only need to import that file. Totally okay to do it this way though

Reply:
Sure.